test/functional/unified-spec-runner/operations.ts

```diff
@@ -173,11 +173,33 @@ operations.set('assertSessionTransactionState', async ({ entities, operation })
   expect(session.transaction.state).to.equal(driverTransactionStateName);
 });
 
+operations.set('assertNumberConnectionsCheckedOut', async ({ entities, operation }) => {
+  const client = entities.getEntity('client', operation.arguments.client);
+  const servers = Array.from(client.topology.s.servers.values());
+  const checkedOutConnections = servers.reduce((count, server) => {
+    const pool = server.s.pool;
+    return count + pool.currentCheckedOutCount;
+  }, 0);
+  process.nextTick(() => {
+    expect(checkedOutConnections).to.equal(operation.arguments.connections);
```

Comment:
Why is this wrapped in nextTick?

Reply:
I can remove that - I was testing a theory.

Reply:
Removed.